chrome/skin/style.css

```diff
@@ -151,3 +151,12 @@ popupset#mainPopupSet panel#PopupAutoCompleteRichResult richlistbox {
 #universal-search-recommendation.highlight #universal-search-recommendation-url {
   color: HighlightText;
 }
+#universal-search-recommendation.highlight #universal-search-recommendation-image {
+  background-color: #FFF;
+  box-shadow: 0;
```

Comment:
What if we just remove the `box-shadow` at line 53? It looks like the same shadow is also applied to the letterbox on line 74, so it shouldn't hurt the letterbox case.